tests/expected/shadow/slices/slice_of_array/test.rs

```diff
@@ -13,7 +13,7 @@ const N: usize = 16;
 static mut SM: kani::shadow::ShadowMem<bool> = kani::shadow::ShadowMem::new(false);
 
 #[kani::proof]
-#[kani::unwind(31)]
+#[kani::unwind(17)]
```

Comment:
Quite a drop too.

Reply:
I'll see whether we can avoid this when moving to CBMC 6.1.1.

Reply:
Reverted with CBMC 6.1.1